tests/Microsoft.Bot.Builder.Ai.LUIS.Tests/LuisRecognizerTests.cs

```diff
@@ -290,20 +290,17 @@ private bool WithinDelta(JToken token1, JToken token2, double delta, bool compar
             return withinDelta;
         }
 
-        private JObject JsonLuisResult(RecognizerResult result)
+        private JObject Json<T>(T result)
```

Comment:
This method is a Clone(). That it's named Json is confusing. 

Also, I don't get why it would return a JObject, if you already have all the strong typing in place.

Reply:
"AsJObject()" or similar would be a less confusing name here.

Reply:
This is solely for testing and it is easier to compare JSon than a strongly typed object.  This is generic test code used for other tests as well.